doc/source/advanced.rst

```diff
@@ -517,6 +517,27 @@ The ``runtime_env`` is a Python dictionary including one or more of the followin
 
   - Example: ``{"OMP_NUM_THREADS": "32", "TF_WARNINGS": "none"}``
 
+- ``container`` (Dict[str, Any]): A dictionary configuring the options for running
+  Ray worker inside a container. This is an experimental feature. Currently Ray requires
+  ``podman`` container runtime and is tested for running container-in-container.
+
+  The following options are required:
+
+  - ``worker_path`` (str): the absolute path for ray's default_worker.py file.
+    You can run ``python -m inspect -d ray.workers.default_worker | grep Origin | cut -d " " -f 2``
+    inside the container to retrieve it.
+  - ``image`` (str): the docker image to run worker inside.
+    Full path like ``docker.io/rayproject/ray-ml:nightly-py38-cpu`` is preferred.
+  
+  To configure the container runtime, you can use ``run_options`` field with a list of command
+  line options (https://docs.docker.com/engine/reference/run/). 
+
+  You can also enable internal configuration ``worker_resource_limits_enabled`` via 
+  ``ray.init(_system_config={"worker_resource_limits_enabled": True})``. This will enforce
+  Ray's per task/actor resource requirement with container's resource limits. In particular,
+  ``--cpu-share, --cpuset-cpus, --memory`` will be added to container spec.
```

Comment:
```suggestion
  ``ray.init(_system_config={"worker_resource_limits_enabled": True})``. This will enforce
  Ray's per-task/actor resource requirement with the container's resource limits. In particular,
  ``--cpu-share, --cpuset-cpus, --memory`` will be added to the container spec.
``` 

This part was a little confusing to me, is it possible to expand on it a little more?  In particular, what happens when this option is off, and how is it different when the option is on?